apps/help-center/help-center-wp-admin.js

```diff
@@ -13,6 +13,10 @@ function AdminHelpCenterContent() {
 	const { setShowHelpCenter } = useDispatch( 'automattic/help-center' );
 	const show = useSelect( ( select ) => select( 'automattic/help-center' ).isHelpCenterShown() );
 	const button = document.getElementById( 'wp-admin-bar-help-center' );
+	const child = button.querySelector( '.ab-item' );
+	if ( child ) {
+		child.classList.add( 'ab-icon' );
+	}
```

Comment:
I think it would be better if you added this directly to the admin bar section. The class is set here ⇢ https://github.com/Automattic/jetpack/blob/trunk/projects/packages/jetpack-mu-wpcom/src/features/help-center/class-help-center.php#L97

The Help Center has two different forms in WP-ADMIN. You would also need to add this bandaid to that other file too.

1. If everything is functioning normal. That is this file you are editing here.
2. In the event that Jetpack is disconnected. That exists in `help-center-wp-admin-disconnected.js`

Reply:
That makes sense! Thanks! As an alternative, I added the `ab-icon` class to the SVG, and it seems to be working well. WDYT?